src/main/java/nextstep/courses/entity/SessionEntity.java

```diff
@@ -0,0 +1,92 @@
+package nextstep.courses.entity;
+
+import nextstep.courses.domain.FreeSession;
+import nextstep.courses.domain.PaidSession;
+import nextstep.courses.domain.Session;
+import nextstep.courses.domain.SessionImage;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+public class SessionEntity {
+    public static final String FREE_SESSION = "FREE";
+    public static final String PAID_SESSION = "PAID";
+    public static final int FREE_FEE = 0;
+    public static final int FREE_MAX_STUDENTS = 0;
+    private Long id;
+    private Timestamp sessionStartAt;
+    private Timestamp sessionEndAt;
+    private String status;
+    private Long imageId;
+    private String sessionType = FREE_SESSION;
+    private Integer sessionFee = FREE_FEE;
+    private Integer maxStudent = FREE_MAX_STUDENTS;
+
+    public SessionEntity() {
+    }
+
+    public SessionEntity(Long id, Timestamp sessionStartAt, Timestamp sessionEndAt, String status, String sessionType, Integer sessionFee, Integer maxStudent) {
+        this.id = id;
+        this.sessionStartAt = sessionStartAt;
+        this.sessionEndAt = sessionEndAt;
+        this.status = status;
+        this.sessionType = sessionType;
+        this.sessionFee = sessionFee;
+        this.maxStudent = maxStudent;
+    }
+
+    public static SessionEntity toEntity(Session session) {
```

Comment:
Entity 에서 호출하는거니 from 은 어떨까요?

```suggestion
    public static SessionEntity from(Session session) {
```

Reply:
맞네요 
수정해서 반영하겠습니다!